sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/_client.py

```diff
@@ -19,13 +19,17 @@
 if TYPE_CHECKING:
     # pylint:disable=unused-import
     from typing import Any, Iterable, Optional, Union
-    from azure.core.credentials import TokenCredential
     from azure.core.paging import ItemPaged
     from azure.core.polling import LROPoller
     from ._models import JsonWebKey
     from ._enums import KeyType
 
 
+def _get_key_id(vault_url, key_name, version=None):
+    without_version = "{}/keys/{}".format(vault_url, key_name)
```

Comment:
Good question -- luckily we know that a KeyClient's vault URL shouldn't have a trailing / because it's [stripped by the base client](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/_shared/client_base.py#L45)